references/proxy-template/pipeline.sh

```diff
@@ -14,21 +14,79 @@
 # limitations under the License.
 
 set -e
-set -x
 
 SCRIPTPATH="$( cd "$(dirname "$0")" || exit >/dev/null 2>&1 ; pwd -P )"
 
+if [ -z "$1" ] || [ "$1" = "--apigeeapi" ];then
+    apiversion="--apigeeapi"
+    TEST_HOST="$APIGEE_ORG-$APIGEE_ENV.apigee.net"
+elif [ -z "$1" ] || [ "$1" = "--googleapi" ];then
```

Comment:
this means that we're not running tests against X/hybird? I'd rather we test against both targets sequentially.

Reply:
sure, I modify the pipeline script